connectors/sources/network_drive.py

```diff
@@ -62,6 +73,26 @@
 LINUX = "linux"
 
 
+class UserAccountDisabledException(Exception):
+    pass
+
+
+class ClientPermissionException(Exception):
+    pass
+
+
+class InvalidLogonHoursException(Exception):
+    pass
+
+
+class PasswordChangeRequiredException(Exception):
+    pass
+
+
+class NoLogonServerException(Exception):
+    pass
```

Comment:
Can we add user-friendly error messages here so that the user can act them?

Reply:
Error message are already set as per the error on line: https://github.com/elastic/connectors/pull/2458/files#diff-29e12700a1a4cf24af22d5bf0a870c9ebfede0ec41dc991406b951d0100393ffR286

Reply:
Ah okay, misread it.